pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>1.609</version>
```

Comment:
As I responded to @amuniz this does not seem to be necessary and so should probably be reverted (unless you can offer an explanation for why you need a newer baseline).